### ISSUE TITLE
[request] Auto select alias

### ISSUE THREAD

**Issue by Carlos-err406** [nextcloud/mail#6235]
### Is your feature request related to a problem? Please describe.

When you are redacting a new email, and write the destinatary's email, if you dont click on the ``+ example@mail.com`` field (nor the ``Enter`` key either), the alias is not considered taken, and send button remains disabled (see screenshots below), and this makes it unconfortable

![screenshot](https://user-images.githubusercontent.com/81443707/163261539-29ecb9fe-6552-4ff7-b4dc-8e81978da645.png)

![screenshot2](https://user-images.githubusercontent.com/81443707/163265865-30a1da29-7a51-404f-b633-3f618f66576c.png)

### Describe the solution you'd like

I'd like it to automatically take the alias, even if you don't click on the ``example@mail.com`` field, since the mail is correctly written, like onfocusout-> addAlias

### Describe alternatives you've considered

maybe add to the field a text like ``Press enter to add``, since is a bit counterintuitive that if you write the whole alias, this is not taken unless you press ``Enter`` or click on the ``+ example@mail.com`` field

**Comment by miaulalala** [comment]
I agree, I have run into this myself. I will add it to https://github.com/nextcloud/mail/issues/6015

**Comment by meichthys** [comment]
Also, to follow up on this, if you paste an email address into the "To" field, the "Send" button is greyed out and there is no indication that the recipient address is not validated.

**Comment by madbob** [comment]
Fixed in 92c1592
